working-open.md

```diff
@@ -0,0 +1,53 @@
+# Working Open
+
+Hypha operates with democratic organizing principles informed by our [values](./values.md). As part of that, we've developed a set of guidelines for how (and exactly what) and why we work in the open.
+
+## Why?
+
+- In order to share and provide opportunities for self- and collective- learning as we work in solidarity with others seeking to create meaningful livelihoods.
```

Comment:
This line is getting at something like "other people can read and learn from our stuff", right? Having feels about [thing-explainer'ing](https://xkcd.com/thing-explainer/) some of this language, but don't want to be pedantic! :)

Wondering if conceiving an "audience" or some personas for this handbook would be helpful. Our parents? Children? A baby boomer investor? Tradespersons?

Reply:
This is our mission/vision statement.

Reply:
I'm going to leave this for now because we are our initial audience and this is our own mission statement! I've added your thoughts to #8